src/wasm_offset_converter.js

```diff
@@ -0,0 +1,82 @@
+function WasmOffsetConverter(wasmBytes) {
+  // This class parses a WASM binary file, and constructs a mapping from
+  // function indices to the start of their code in the binary file.
+  //
+  // The main purpose of this module is to enable the conversion of function
+  // index and offset from start of function to an offset into the WASM binary.
+  // This is needed to look up the WASM source map as well as generate
+  // consistent program counter representations given v8's non-standard
+  // WASM stack trace format.
+
+  var offset = 8;
```

Comment:
i am having a hard time understanding this code. perhaps naming this more specifically might help (is it a function offset? binary offset?), or adding a comment. likewise count and idx.